Library/Homebrew/brew.sh

```diff
@@ -194,6 +194,17 @@ EOS
       odie <<EOS
 You have not agreed to the Xcode license. Please resolve this by running:
   sudo xcodebuild -license accept
+EOS
+    fi
+    if [[ "$XCRUN_STATUS" -ne 0 && "$XCRUN_OUTPUT" = *"invalid active developer path"* ]]
```

Comment:
Would be good to share the `"$XCRUN_STATUS" -ne 0` with above

Reply:
Normally I'd agree, but I'm not entirely sure that it looks better / simpler in this case.
In a less verbose language that didn't use up a line for `then` / `fi`, sure. In bash, perhaps not.

Added the change in 777f258. I'll let you make a call on whether it's good.